backend/controller/products-controller.js

```diff
@@ -0,0 +1,17 @@
+import  {Product} from "../models/productModel.js";
+
+const getProducts = async(req ,res) => {
```

Comment:
Can you explain the motivation of a controller? if we already have the route defined and it's effectively the same thing

Reply:
as I thought for further there will be more routes and function call will be added so its better to make them in a seperate file.